cea/technologies/thermal_network/thermal_network_costs.py

```diff
@@ -136,30 +135,41 @@ def calc_Ctot_cooling_plants(network_info):
         Opex_fixed_chiller = 0
         Capex_CT = 0
         Opex_fixed_CT = 0
-        print plant_heat_peak_kW
         if plant_heat_peak_kW > 0: # we have non 0 demand
             peak_demand = plant_heat_peak_kW * 1000  # convert to W
             # calculate Heat loss costs
             # Clark D (CUNDALL). Chiller energy efficiency 2013.
             # FIXME: please add reference in the documentation of this function (see thermal_network_matrix)
             print 'Calculating cost of heat production at plant: ', plant_number
-            # Read in building demand
-            disconnected_demand = {}
-            for building in network_info.building_names:
-                disconnected_demand[building] = pd.read_csv(
-                    network_info.locator.get_demand_results_file(building))
-            for t in range(HOURS_IN_YEAR):
-                # calculate COP of plant operation in this hour based on supplied loads
-                # calculate plant COP according to the cold water supply temperature in SG context
-                supplied_systems = find_non_zero_demand_systems(network_info, t, disconnected_demand)
+            if network_info.config.yearly_cost_calculations:
+                supplied_systems = find_systems_string(network_info.full_cooling_systems)
```

Comment:
why not using line 158-160 to have the exact supplied_systems?

Reply:
I think this has resolved itself as the cose has chenged since the comment? At least I can't ind this section anymore.